client/register-businesses.py

```diff
@@ -120,8 +135,12 @@ def shop_owners():
         client.await_block()
 
     create_offerings(cid, 5)
-    offerings_ipfs_cids = Ipfs.add_recursive_multiple(glob.glob(OFFERINGS_PATH + '/*.json'))
-    print(f'Uploaded offerings to ipfs: ipfs_cids: {offerings_ipfs_cids}')
+    if args.ipfs_local:
```

Comment:
We can save quite some lines if we consolidate the ipfs add methods to one method:

```
def add_multiple(files, local = false): # set default value of local to false.
      # extract env_vars if not local and use the remote version

      # use the local version if local.
   
```

Reply:
done

Reply:
and false is not a reserved keyword in python, it's False ;)